### PR TITLE
adding custom external link for opening new windows

### PR DESCRIPTION
**Pull request by readeral**

### CODE REVIEW

File: src/components/block-content.tsx [hunk 1]
```diff
@@ -35,6 +35,31 @@ InternalLink.propTypes = {
...
+    _key: string;
+    _type: string;
+    blank: boolean;
+    href: string;
+  };
+  children: ReactNode;
+};
+
+const ExternalLink: FC<ExternalLinkProps> = ({children, mark}) => {
+  const reference = {...mark, children: children[0], isInternal: false};
```
Comment (BarryThePenguin) [inline src/components/block-content.tsx]:
```suggestion
  const reference = {...mark, children, isInternal: false};
```

Is it ok to just pass `children` through? Unsure why we need to access the first element? 🤔

File: src/components/link.tsx [hunk 2]
```diff
@@ -32,19 +32,26 @@ const Link: FC<LinkProps> = ({as, href, isInternal, ...props}) => {
     );
   }
 
+  if (blank)
+    return (
+      <ThemeUiLink href={href} target="_blank" rel="noopener" {...props} />
```
Comment (BarryThePenguin) [inline src/components/link.tsx]:
```suggestion
      <ThemeUiLink href={href} target="_blank" rel="noreferrer noopener" {...props} />
```

File: src/components/block-content.tsx (hunk 1 above)
Comment (readeral) [reply src/components/block-content.tsx]: Seems this is true!

File: src/components/link.tsx (hunk 2 above)
Comment (readeral) [reply src/components/link.tsx]: Ta

**Pull request merged**